app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
```

Comment:
Potentially. Options to fix this are:

* add JS to the page, which feels like overkill at this stage.
* move the export button to the bottom of the page

you have a preference?